code/upaya/webpack.config.js

```diff
@@ -0,0 +1,35 @@
+var webpack = require('webpack');
+
+module.exports = {
+  entry: './public/js/entry.js',
+
+  output: {
+    filename: 'bundle.js',
+    path: __dirname + '/public/js'
```

Comment:
What do you feel about structuring this in /public/js/generated/{dev,prod}/bundle.js?

It keeps the generated code separate from the source code.